### DISCUSSION TITLE
keyball44 LEDランドが剥離してしまった

### DISCUSSION THREAD

**Discussion by kr545** [Yowkees/keyball#620] [Q&A]
お世話になっております。
keyball44のＬＥＤランドを剥がしてしまい接続先がわからず困っています。
剥がれたのは赤色と青色で示している８番の右側二つなのですが、どのようにすればＬＥＤを修復できるかご教授していただけると幸いです。

![IMG_3017](https://github.com/Yowkees/keyball/assets/97976292/7be0e64d-a942-438d-bd80-875b5e13e727)

**Comment by Ryo624** [comment]
下記画像の同色どうしをエナメル線かゴム被覆線でつなぐと復活できると思われます。
![346352164-7be0e64d-a942-438d-bd80-875b5e13e727](https://github.com/Yowkees/keyball/assets/143698349/9da33d3f-d057-4c5d-8c76-aa8ee5b129fb)

**Reply by kr545** [reply]
ありがとうございました！

無事にすべてのＬＥＤを点灯させることができました。

また大変恐縮なのですが、今回ＬＥＤを修復する過程でトラックボール側のOLEDがうまく作動しなくなってしまったので追加で質問させてください。

症状としましては、
○トラックボール側にＵＳＢを接続するとＯＬＥＤが点灯する時と点灯しないときがある。
○トラックボール側を親にしてＴＲＲＳケーブルを反対側の基板に接続するとキーボードの認識がうまくいかない。
○反対側の基板を親にしてトラックボール側をＴＲＲＳケーブルでつなげると問題なく動作する。

ほかのdiscussionで似たような質問があったので、トラックボール側のＴＲＲＳケーブル周辺のはんだづけ不良だと思うのですが、半田付けをやりなおしても症状が改善しないので可能であればアドバイスをいただけると幸いです。

よろしくお願いいたします。
![18F1287D-52EE-4FA3-B146-FD21E6A55EED_4_5005_c](https://github.com/Yowkees/keyball/assets/97976292/c7fdc670-ffa5-4e99-aae5-3b1cfdd42d92)
![A3F45988-B868-476F-BE9E-0A1B86736CFE_4_5005_c](https://github.com/Yowkees/keyball/assets/97976292/6ef8ab01-bad2-4888-8838-ef76bae0118b)
